packages/engine/Source/Core/createWorldBathymetryAsync.js

```diff
@@ -0,0 +1,48 @@
+import CesiumTerrainProvider from "./CesiumTerrainProvider.js";
+import defaultValue from "./defaultValue.js";
+
+/**
+ * Creates a {@link CesiumTerrainProvider} instance for the {@link https://cesium.com/content/#cesium-world-bathymetry | Cesium World Bathymetry}.
+ *
+ * @function
+ *
+ * @param {Object} [options] Object with the following properties:
+ * @param {Boolean} [options.requestVertexNormals=false] Flag that indicates if the client should request additional lighting information from the server if available.
+ * @param {Boolean} [options.requestWaterMask=false] Flag that indicates if the client should request per tile water masks from the server if available.
```

Comment:
I don't believe a watermask exists for CWB. I'm confirming now. If that's the case, let's omit this option (and update the example below)

Reply:
@jjspace Just confirmed there's no watermask. Let's remove the option.